fhirr4/ballerina/src/main/resources/fhirservice/tests/test.bal

```diff
@@ -0,0 +1,276 @@
+// Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
```

Comment:
Let's rename to test_fhir_service.bal or something appropriate.

Reply:
Renamed to fhir_service_test.bal.